spec/expression.dd

```diff
@@ -1100,6 +1100,66 @@ $(GNAME CastExpression):
         (foo) - p;      // subtract p from foo
         -------------
 
+$(H4 $(LNAME2 cast_basic_data_types, Basic Data Types))
+    $(P Cast between different types of data will not be performed automatically.
+        The next operation generates the following error:
+        "cannot implicitly convert expression a of type int to byte".
+    )
+
+        -------------
+        int a;
+        byte b = a;
+        -------------
+
+    $(P In the following example, the cast operation will truncate the number.)
+
+        $(SPEC_RUNNABLE_EXAMPLE_RUN
+            -------------
+            int a = 64389; // 00000000 00000000 11111011 10000101
+            byte b = cast(byte) a;       // 10000101
+            ubyte c = cast(ubyte) a;     // 10000101
+            short d = cast(short) a;     // 11111011 10000101
+            ushort e = cast(ushort) a;   // 11111011 10000101
+
+            writeln(b);
+            writeln(c);
+            writeln(d);
+            writeln(e);
+            -------------
+        )
+
+    $(P In this case, the sign will be extended.)
```

Comment:
Please add a generalizing statement. I.E: "when casting an integral type from a narrower type to a wider type ... bla bla ... sign extension ... bla bla ".

Do not add meaningless paragraphs like this one.